docs/getting-started/register-and-serve-features-training-sets.mdx

```diff
@@ -39,6 +39,19 @@ class User:
   age = ff.Feature(dataset[["user", "top_item", "timestamp"]], variant="simple", type=ff.Int, inference_store=redis)
 ```
 
+#### Registering Multiple Features
```

Comment:
Should specify `Registering Features In Batch`

Reply:
Should I change "Registering Multiple Features" to "Registering Features In Batch"?